internal/codespaces/codespaces.go

```diff
@@ -18,7 +18,7 @@ func connectionReady(codespace *api.Codespace) bool {
 		codespace.State == api.CodespaceStateAvailable
 }
 
-type apiClient interface {
+type LiveshareApiClient interface {
```

Comment:
nit: this client is not talking to the liveshare service but github instead. I'd perhaps name it `GitHubAPIClient`.